### ISSUE TITLE
ca_cert_identifier doesn't added to replica instance

### ISSUE THREAD

**Issue by navi86** [terraform-providers/terraform-provider-aws#11546]
ca_cert_identifier doesn't work for replica db

### Terraform Version

Terraform v0.12.12

### Affected Resource(s)

* aws_2.43.0

### Terraform Configuration Files

```hcl
resource "aws_db_instance" "source" {
  identifier              = "rds-source"
  engine                  = "postgres"
  allocated_storage  = 5
  name                    = "test"
  username                = "admin"
  password                = "1qaz@WSX"
  skip_final_snapshot     = true
  ca_cert_identifier      = "rds-ca-2019"
  apply_immediately       = true
}

resource "aws_db_instance" "replica" {
  identifier            ="rds-replica"
  replicate_source_db   = "${aws_db_instance.source[0].arn}"
  skip_final_snapshot   = true
  ca_cert_identifier    = "rds-ca-2019"
  apply_immediately       = true
}
```

### Expected Behavior
replica rds server has "rds-ca-2019" as source db instance

### Actual Behavior

replica rds server has "rds-ca-2015"

### Steps to Reproduce

1. `terraform apply`

**Comment by bflad** [comment]
Relevant pull requests: https://github.com/terraform-providers/terraform-provider-aws/pull/11541 (/ https://github.com/terraform-providers/terraform-provider-aws/pull/11547 (closed duplicate))

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 2.45.0 of the Terraform AWS Provider, Thursday next week. 👍
